x/globalfee/ante/antetest/fee_test.go

```diff
@@ -468,8 +468,7 @@ func (s *IntegrationTestSuite) TestGlobalFeeMinimumGasFeeAnteHandler() {
 			txCheck:         true,
```

Comment:
Perhaps adding some tests using multiple bypass msg in one tx could be an improvement.

Reply:
Also a test that shows the how the result of mixing bypass messages and non-bypass mesagges.

e.g.
combining normal Txs with any of `["/ibc.core.channel.v1.MsgRecvPacket", "/ibc.core.channel.v1.MsgAcknowledgement","/ibc.applications.transfer.v1.MsgTransfer", "/ibc.core.channel.v1.MsgTimeout", "/ibc.core.channel.v1.MsgTimeoutOnClose"]`

This can be followed up on in another PR if it is not convenient to do here

Reply:
good idea , i added it here as todo later https://github.com/cosmos/gaia/issues/2213. so that this pr can merged early to unblock us.